ROADMAP.md

```diff
@@ -28,21 +18,19 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
   * Minor version upgrades [#2532](https://github.com/rook/rook/issues/2532)
 * Ceph
   * Ceph-CSI plug-in integration [#1385](https://github.com/rook/rook/issues/1385)
```

Comment:
A general statement about beta-level CSI plugin integration would be sufficient. I don't think we need to list these specific csi issues.

Reply:
Ah, I see now that's what your second commit did